bftengine/src/bftengine/RequestHandler.cpp

```diff
@@ -108,7 +108,7 @@ void RequestHandler::execute(IRequestsHandler::ExecutionRequestsQueue& requests,
         const auto tick = Tick{payload.component_id, req.executionSequenceNum};
         (*cron_table_registry_)[payload.component_id].evaluate(tick);
       } else {
-        LOG_ERROR(GL, "Received a Tick, but the cron table registry is not initialized");
```

Comment:
I would say this might be an actual error depending if the cron table is really optional or not. As far as I can see an empty cron table is constructed by default in a Replica so it would be very difficult for this object to be missing in normal circumstances.